modules/new_profile/php/profile.class.inc

```diff
@@ -120,8 +135,126 @@ class Profile extends \NDB_Page
                 ->withStatus(405)
                 ->withHeader("Allow", "GET,POST");
         }
+    }
 
+    /**
+     * Processes the values and saves to database
```

Comment:
This function comment isn't accurate. Can you update it? Also the params and return values could use more detail.